src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -11,71 +11,88 @@ export interface ShippingInformationProps {
 }
 
 const ArtworkSidebarShippingInformation: React.FC<ShippingInformationProps> = ({
-  artwork: { shippingOrigin, shippingInfo, isUnlisted },
+  artwork: {
+    isUnlisted,
+    priceIncludesTaxDisplay,
+    shippingOrigin,
+    shippingInfo,
+    taxInfo,
+  },
 }) => {
   const { t } = useTranslation()
   const { trackEvent } = useTracking()
 
   if (isUnlisted) {
     return (
       <>
-        {shippingOrigin && (
+        {!!shippingOrigin && (
           <Text variant="sm" color="black60">
             {t`artworkPage.sidebar.shippingAndTaxes.shipsFrom`} {shippingOrigin}
           </Text>
         )}
 
-        <Text variant="xs" color="black60">
-          Shipping and taxes may apply at checkout.{" "}
-          <RouterLink
-            inline
-            to="https://support.artsy.net/s/article/How-are-taxes-customs-VAT-and-import-fees-handled-on-works-listed-with-secure-checkout"
-            target="_blank"
-            rel="noopener noreferrer"
-            onClick={() => {
-              const payload: ClickedOnLearnMore = {
-                action: ActionType.clickedOnLearnMore,
-                context_module: "Sidebar",
-                subject: "Learn more",
-                type: "Link",
-                flow: "Shipping",
-              }
+        {!!taxInfo && (
```

Comment:
I assumed that we also only want to display this line if tax info is present - similar to how we do this for listed works. Is that not the case?

Reply:
Ohh I don't feel confident on weighing on this one. I remember Peter spent quite some time discussing this with legal team before making a call about the rules for displaying shipping and tax copies. It might be worth to double check these with Jini.